js/myparcel.js

```diff
@@ -142,11 +160,11 @@ MyParcel = {
      */
     defaultCheckCheckbox: function(selectedOption){
 		if(selectedOption == 'mypa-only-recipient'){
-            $('#mypa-only-recipient-selector').prop('checked', true);
-            $('#mypa-only-recipient-selector').prop({disabled: true});
+            $('#mypa-only-recipient-selector').prop('checked', true).prop({disabled: true});
+            $('#mypa-recipient-only').html(' (Inclusief)');
 		} else {
-            $('#mypa-only-recipient-selector').prop('checked', false);
-            $('#mypa-only-recipient-selector').removeAttr("disabled");
+            $('#mypa-only-recipient-selector').prop('checked', false).removeAttr("disabled");
+            $('#mypa-recipient-only').html(' ( + &euro;' + this.data.config.priceOnlyRecipient +')');
```

Comment:
We gebruiken nu 2 termen die hetzelfde betekend:
only-recipient
recipient-only

Je moet even in de api-doc opzoeken wat we als standaard gebruiken, en dan dit even aanpassen.